<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleCitationBibliographyWidget</class>
 <widget class="QWidget" name="SimpleCitationBibliographyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>102</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolButton" name="addCitation">
     <property name="toolTip">
      <string>Insert citation at current position</string>
     </property>
     <property name="text">
      <string>Insert Citation</string>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FormattingButton" name="addBibliography">
     <property name="text">
      <string>Insert Bibliography</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="configureBibliography">
     <property name="toolTip">
      <string>Configure bibliography</string>
     </property>
     <property name="text">
      <string>Configure Bibliography</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="SpecialSpacer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormattingButton</class>
   <extends>QToolButton</extends>
   <header>dialogs/FormattingButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
